### ISSUE TITLE
Inspect npm dependencies

### ISSUE THREAD

**Issue by aldnav** [emanlodovice/youtube-dl-desktop#8]
As of writing,  when running `npm install` yields the following:

```shell
Installed to /Users/aldnav/Projects/youtube-dl-desktop/web/node_modules/node-sass/vendor/darwin-x64-67/binding.node
added 961 packages from 646 contributors and audited 7673 packages in 180.389s
found 204 vulnerabilities (1 low, 200 moderate, 3 high)
  run `npm audit fix` to fix them, or `npm audit` for details
```

Steps to replicate:
```
$ cd web
$ npm install
```

Environment:
```shell
$ npm -v  # 6.5.0
$ node -v  # v11.9.0
```

Just putting it out here for vulnerability checking.

**Comment by arnellebalane** [comment]
These are the available updates for the npm packages used in this project:

```
Minor Update New backwards-compatible features.
- vue                           2.5.16  ❯  2.6.6   https://github.com/vuejs/vue#readme
- cross-env devDep              5.1.6   ❯  5.2.0   https://github.com/kentcdodds/cross-env#readme
- node-sass devDep              4.9.0   ❯  4.11.0  https://github.com/sass/node-sass
- vue-template-compiler devDep  2.5.16  ❯  2.6.6   https://github.com/vuejs/vue/tree/dev/packages/vue-template-compiler#readme

Major Update Potentially breaking API changes. Use caution.
- babel-loader devDep        7.1.4    ❯  8.0.5   https://github.com/babel/babel-loader
- css-loader devDep          0.28.11  ❯  2.1.0   https://github.com/webpack-contrib/css-loader
- file-loader devDep         1.1.11   ❯  3.0.1   https://github.com/webpack-contrib/file-loader
- sass-loader devDep         6.0.7    ❯  7.1.0   https://github.com/webpack-contrib/sass-loader
- vue-loader devDep          13.7.2   ❯  15.6.4  https://github.com/vuejs/vue-loader
- webpack devDep             3.12.0   ❯  4.29.5  https://github.com/webpack/webpack
- webpack-dev-server devDep  2.11.2   ❯  3.1.14  https://github.com/webpack/webpack-dev-server#readme
```

**Comment by arnellebalane** [comment]
@emanlodovice @aldnav I'll see if I can upgrade these packages to their latest versions 👍

**Comment by arnellebalane** [comment]
@emanlodovice @aldnav Fixed in #9 

```
added 974 packages from 422 contributors and audited 9063 packages in 13.407s
found 0 vulnerabilities
```
